app/models/file.ts

```diff
@@ -1,13 +1,28 @@
 import { attr, belongsTo, hasMany } from '@ember-decorators/data';
 import DS from 'ember-data';
+import { Link } from 'jsonapi-typescript';
+
+import getHref from 'ember-osf-web/utils/get-href';
 
 import BaseFileItem from './base-file-item';
 import CommentModel from './comment';
 import FileVersionModel from './file-version';
 import NodeModel from './node';
+import { OsfLinks } from './osf-model';
 import UserModel from './user';
 
+export interface FileLinks extends OsfLinks {
+    html: never;
+    info: Link;
```

Comment:
We could add the `render` link here if you'd like, otherwise we can wait until we mirage it up and/or platform and services get it in.